addons/web/static/src/scss/webclient.scss

```diff
@@ -11,6 +11,10 @@
     background-color: $o-webclient-background-color;
 }
 
+img.rounded-circle {
+    object-fit: cover;
+}
```

Comment:
It should be done case by case like you have done in the other modules, this rule is too generic and will have unwanted side effects.

Reply:
> It should be done case by case like you have done in the other modules, this rule is too generic and will have unwanted side effects.

@seb-odoo thank bro. Done